### ISSUE TITLE
Design Improvement Needed in Contribition.html

### ISSUE THREAD

**Issue by fineanmol** [fineanmol/Hacktoberfest2k21#189]
Hello Guys, New Approach or designs are appreciated in Contribution.html
[Preview here](https://fineanmol.github.io/Hacktoberfest2k21/Contributors.html)

example:  The dark theme can be added with a toggle switch. 
Card designs can be improved.

**Comment by SukritSethi** [comment]
Hi, I am pretty new to web dev. But I think I can pull this off in 3-4 days. Could u assign me with the dark theme task of the page.

**Comment by fineanmol** [comment]
Hi @SukritSethi , We are glad you want to contribute in this repo. Please fork this repo, made your changes and make a pull request. 
We would love to see your work !

**Comment by RahulRudra81** [comment]
I am too a newbie but I think I could pull this off and also learn some good stuff . Can I  work on this?

**Comment by fineanmol** [comment]
> I am too a newbie but I think I could pull this off and also learn some good stuff . Can I work on this?

Yes Please, Do your contributions and pull a request.

**Comment by Naman-pokhriyal** [comment]
I think the hover transition, card size, theme switch and few other things can be better.  Can I work on this?

**Comment by fineanmol** [comment]
> I think the hover transition, card size, theme switch and few other things can be better. Can I work on this?

Yes Naman, Please show you magic ✨

**Comment by tuminzee** [comment]
Hello I think a rounded image of our GitHub avatar would be more suitable for the page.
Please tell me what you think about it.

![image](https://user-images.githubusercontent.com/58180803/135706111-7941c165-c58e-489c-b25c-9fed929e8484.png)

**Comment by fineanmol** [comment]
> Hello I think a rounded image of our GitHub avatar would be more suitable for the page. Please tell me what you think about it.
> 
> ![image](https://user-images.githubusercontent.com/58180803/135706111-7941c165-c58e-489c-b25c-9fed929e8484.png)

Yes, Its looking lovely

**Comment by tuminzee** [comment]
> > Hello I think a rounded image of our GitHub avatar would be more suitable for the page. Please tell me what you think about it.
> > ![image](https://user-images.githubusercontent.com/58180803/135706111-7941c165-c58e-489c-b25c-9fed929e8484.png)
> 
> Yes, Its looking lovely
 
#389

**Comment by fineanmol** [comment]
> > > Hello I think a rounded image of our GitHub avatar would be more suitable for the page. Please tell me what you think about it.
> > > ![image](https://user-images.githubusercontent.com/58180803/135706111-7941c165-c58e-489c-b25c-9fed929e8484.png)
> > 
> > 
> > Yes, Its looking lovely
> 
> #389

Hi @tuminzee , Layout of the CSS has been conflicted, List view needs to be shown in Grid. Please try to contribute there.

**Comment by RahulRudra81** [comment]
@fineanmol I added my name in the contributors list and you merged it 2days ago but I can't see my name there now !! Can I add it again??

**Comment by fineanmol** [comment]
> @fineanmol I added my name in the contributors list and you merged it 2days ago but I can't see my name there now !! Can I add it again??

Yes please. Maybe some other contributor removed your name by mistake.

**Comment by swatisingh46** [comment]
Can you assign this issue to me
